py3status/modules/sysdata.py

```diff
@@ -10,19 +10,38 @@
         (default 75)
     med_threshold: percent to consider CPU or RAM usage as 'medium load'
         (default 40)
+    padding: length of space padding to use on the left
+        (default 0)
+    precision: precision of values
+        (default 2)
+    zone: thermal zone to use. If None try to guess CPU temperature
+        (default None)
 
 Format placeholders:
     {cpu_temp} cpu temperature
     {cpu_usage} cpu usage percentage
+    {load1} load average over the last minute
+    {load5} load average over the five minutes
+    {load15} load average over the fifteen minutes
     {mem_total} total memory
     {mem_used} used memory
     {mem_used_percent} used memory percentage
+    {swap_total} total swap
+    {swap_used} used swap
+    {swap_used_percent} used swap percentage
 
 Color options:
     color_bad: Above high_threshold
     color_degraded: Above med_threshold
     color_good: Below or equal to med_threshold
 
+Color conditionals:
+    cpu: change color based on the value of cpu_usage
+    load: change color based on the value of load1
+    mem: change color based on the value of mem_used_percent
+    temp: change color based on the value of cpu_temp
+    swap: change color based on the value of swap_used_percent
```

Comment:
Hmmm,  I think we can do things in a more transparent way for the user eg `{mem_used}` would display in the correct color automatically.  But we do have the issue about how we provide the flexibility to the user.

I think this needs some thought.